r/data-raw/codegen.R

```diff
@@ -0,0 +1,160 @@
+# pak::pkg_install("romainfrancois/decor")
```

Comment:
When does data-raw/codegen.R get called? Can you please update the README with developer instructions (and if necessary/appropriate, add Suggests dependencies to DESCRIPTION)?

Reply:
Sure. Will do that when I'm sure this is solid.